API.md

```diff
@@ -400,6 +400,23 @@ Access: read only.
 Contains the default authentication configuration if a default strategy was set via
 [`server.auth.default()`](#server.auth.default()).
 
+#### <a name="server.caches" /> `server.caches`
+
+Access: read only.
+
+An object containing the cache provisions, where each key is a cache name and the values are objects with the following properties:
+
+- `segments` - an object where each key is a segment name and the values are [cache policies](https://hapi.dev/family/catbox/api#policy) created with [server.cache()](#server.cache()).
+- `shared` - indicates if the cache provision allows shared segments.
+
+```js
+const Hapi = require('@hapi/hapi');
+const server = Hapi.server({ port: 80 });
+const cache = server.cache({ segment: 'countries', expiresIn: 60 * 60 * 1000 });
+
+console.log(server.caches['_default'].segments.countries === cache);  // true
```

Comment:
I believe this is the first time that the name of the default cache has been public.  If we want to give it a different name or expose a symbol for it, now might be a good time.

Reply:
@hapijs/tsc any thoughts on that?

Reply:
I agree with @devinivy here.

Reply:
I'm fine with `_default`. It is already a reserved name since you can't provision a cache with such a name.

Reply:
In my opinion the underscore doesn't just imply that it's internal, but that it's actually private.

Reply:
So, by @kanongil comment, renaming `_default` may actually lead to a breaking change because an application could already have such name in use. Using a symbol might be a way to avoid that. 

@devinivy I am assuming that your suggestion would be to use an internal symbol, which we would have to expose in order to somehow "serialize" it's name right?

Reply:
I believe that's what @devinivy meant.